crates/re_data_ui/src/component.rs

```diff
@@ -85,8 +93,20 @@ impl DataUi for EntityComponentWithInstances {
                 .body(|mut body| {
                     re_ui::ReUi::setup_table_body(&mut body);
                     let row_height = re_ui::ReUi::table_line_height();
-                    body.rows(row_height, num_instances, |index, mut row| {
-                        if let Some(instance_key) = instance_keys.get(index) {
+                    body.rows(row_height, max_row, |index, mut row| {
+                        if index == max_row - 1 && num_instances > max_row {
+                            // last row, suggest that there is more.
+                            row.col(|ui| {
+                                ui.label(format!(
+                                    "{} more…",
```

Comment:
I think this will look better, but maybe not.

```suggestion
                                    "… and {} more.",
```

btw, nice use of `format_large_number`! It's easy to forget to use that one, so nice that you didn't :)

Reply:
TIL 🤣

Not only I would have *totally* forgotten that, but I didn't even realise that Copilot added it for me! (In fairness, I was in a bit of a rush then.)

Reply:
I like your version more, but without the space after the `…`. (Couldn't find any indications online that there ever should be one.)